### ISSUE TITLE
fully offline/local?

### ISSUE THREAD

**Issue by 0lm** [Spafbi/simplified-miscreated-server-setup#18]
Hello. I did not run the script yet. Before I do, I wanted to ask if flly local playing is possible. It is really nice to automate server creation! But personally, I would love to play Solo, too. For that, I don't need to make my server public or my router to open ports (which owuld make it vulnerable). So, does this script also gives the choice to run the game fully offline in my own network/localhost, without changing settings in my firewall and router, that makes it public?

**Comment by Therosin** [comment]
Yes it does, you still need a working network connection as this is used to authorize the server to validate game/map ownership etc
but if you read the wiki you will see that you can leave your ports closed. and use the listed methods to direct connect to your locally hosted server
